src/installer/windows/index.ts

```diff
@@ -71,8 +72,15 @@ export class WindowsToolchainInstaller extends VerifyingToolchainInstaller<Windo
     }
     core.debug(`Swift installed at "${swiftPath}"`)
     const visualStudio = await VisualStudio.setup(this.vsRequirement)
-    await visualStudio.update(installation.sdkroot)
-    const swiftFlags = `-sdk %SDKROOT% -I %SDKROOT%/usr/lib/swift -L %SDKROOT%/usr/lib/swift/windows`
+    await visualStudio.update(sdkroot)
+    const swiftFlags = [
+      '-sdk',
+      `'${sdkroot}'`,
+      '-I',
+      `'${path.join(sdkroot, 'usr', 'lib', 'swift')}'`,
```

Comment:
This feels unnecessary and will only lead to one more line of code (I tried to make it into one line but that's impossible). And in fact I didn't come up with a perfect naming of the variable, so it might be better kept as-is to avoid further confusion (in the production code).